packages/kbn-test/src/functional_test_runner/lib/snapshots/decorate_snapshot_ui.ts

```diff
@@ -161,6 +161,7 @@ function getSnapshotState(file: string, updateSnapshot: SnapshotUpdateState) {
     path.join(dirname + `/__snapshots__/` + filename.replace(path.extname(filename), '.snap')),
     {
       updateSnapshot,
+      // @ts-expect-error
```

Comment:
Why is this error expected? I feel like we probably need to update parameters here